diff --git a/static/images/about-bg.jpg b/static/images/about-bg.jpg
old mode 100755
new mode 100644
diff --git a/static/images/contact-bg.jpg b/static/images/contact-bg.jpg
old mode 100755
new mode 100644
diff --git a/static/images/home-bg.jpg b/static/images/home-bg.jpg
old mode 100755
new mode 100644
diff --git a/static/images/post-bg.jpg b/static/images/post-bg.jpg
old mode 100755
new mode 100644
diff --git a/templates/page.html b/templates/page.html
index 2414549..5c9d26c 100644
--- a/templates/page.html
+++ b/templates/page.html
@@ -2,32 +2,30 @@
 
 {% block title %}{{ page.title }}{% endblock title %}
 
-{# <!-- Choosing cover image --> #}
-{% if page.cover %}
-  {% if page.cover|lower|truncate(4, True, '') == "http" %}
-    {% set selected_cover = page.cover %}
-  {% else %}
-    {% set selected_cover = SITEURL+"/"+page.cover %}
-  {% endif %}
-{% elif page.og_image %}
-  {% if page.og_image|lower|truncate(4, True, '') == "http" %}
-    {% set selected_cover = page.og_image %}
-  {% else %}
-    {% set selected_cover = SITEURL+"/"+page.og_image %}
-  {% endif %}
-{% elif HEADER_COVER %}
-  {% if HEADER_COVER|lower|truncate(4, True, '') == "http" %}
-    {% set selected_cover = HEADER_COVER %}
-  {% else %}
-    {% set selected_cover = SITEURL+"/"+HEADER_COVER %}
-  {% endif %}
-{% elif page.color %}
-  {% set selected_color = page.color %}
-{% elif HEADER_COLOR %}
-  {% set selected_color = HEADER_COLOR %}
-{% endif %}
-
 {% block header %}
+    {% if page.cover %}
+        {% if page.cover|lower|truncate(4, True, '') == "http" %}
+            {% set selected_cover = page.cover %}
+        {% else %}
+            {% set selected_cover = SITEURL+"/"+page.cover %}
+        {% endif %}
+        {% elif page.og_image %}
+        {% if page.og_image|lower|truncate(4, True, '') == "http" %}
+            {% set selected_cover = page.og_image %}
+        {% else %}
+            {% set selected_cover = SITEURL+"/"+page.og_image %}
+        {% endif %}
+        {% elif HEADER_COVER %}
+        {% if HEADER_COVER|lower|truncate(4, True, '') == "http" %}
+            {% set selected_cover = HEADER_COVER %}
+        {% else %}
+            {% set selected_cover = SITEURL+"/"+HEADER_COVER %}
+        {% endif %}
+        {% elif page.color %}
+        {% set selected_color = page.color %}
+        {% elif HEADER_COLOR %}
+        {% set selected_color = HEADER_COLOR %}
+    {% endif %}
     <!-- Page Header -->
     <!-- Set your background image for this header on the line below. -->
     <header id="post-header" class="has-cover">
diff --git a/tests/content/assets/images/article_cover.jpg b/tests/content/assets/images/article_cover.jpg
deleted file mode 120000
index 22ce499..0000000
--- a/tests/content/assets/images/article_cover.jpg
+++ /dev/null
@@ -1 +0,0 @@
-cover.jpg
\ No newline at end of file
diff --git a/tests/content/assets/images/article_cover.jpg b/tests/content/assets/images/article_cover.jpg
new file mode 100644
index 0000000..e43b6d1
Binary files /dev/null and b/tests/content/assets/images/article_cover.jpg differ
diff --git a/tests/content/assets/images/arul_author_cover.jpg b/tests/content/assets/images/arul_author_cover.jpg
deleted file mode 120000
index 22ce499..0000000
--- a/tests/content/assets/images/arul_author_cover.jpg
+++ /dev/null
@@ -1 +0,0 @@
-cover.jpg
\ No newline at end of file
diff --git a/tests/content/assets/images/arul_author_cover.jpg b/tests/content/assets/images/arul_author_cover.jpg
new file mode 100644
index 0000000..e43b6d1
Binary files /dev/null and b/tests/content/assets/images/arul_author_cover.jpg differ
diff --git a/tests/content/assets/images/bar_category_cover.jpg b/tests/content/assets/images/bar_category_cover.jpg
deleted file mode 120000
index 22ce499..0000000
--- a/tests/content/assets/images/bar_category_cover.jpg
+++ /dev/null
@@ -1 +0,0 @@
-cover.jpg
\ No newline at end of file
diff --git a/tests/content/assets/images/bar_category_cover.jpg b/tests/content/assets/images/bar_category_cover.jpg
new file mode 100644
index 0000000..e43b6d1
Binary files /dev/null and b/tests/content/assets/images/bar_category_cover.jpg differ
diff --git a/tests/content/assets/images/bar_tag_cover.jpg b/tests/content/assets/images/bar_tag_cover.jpg
deleted file mode 120000
index 22ce499..0000000
--- a/tests/content/assets/images/bar_tag_cover.jpg
+++ /dev/null
@@ -1 +0,0 @@
-cover.jpg
\ No newline at end of file
diff --git a/tests/content/assets/images/bar_tag_cover.jpg b/tests/content/assets/images/bar_tag_cover.jpg
new file mode 100644
index 0000000..e43b6d1
Binary files /dev/null and b/tests/content/assets/images/bar_tag_cover.jpg differ
diff --git a/tests/content/assets/images/foo_category_cover.jpg b/tests/content/assets/images/foo_category_cover.jpg
deleted file mode 120000
index 22ce499..0000000
--- a/tests/content/assets/images/foo_category_cover.jpg
+++ /dev/null
@@ -1 +0,0 @@
-cover.jpg
\ No newline at end of file
diff --git a/tests/content/assets/images/foo_category_cover.jpg b/tests/content/assets/images/foo_category_cover.jpg
new file mode 100644
index 0000000..e43b6d1
Binary files /dev/null and b/tests/content/assets/images/foo_category_cover.jpg differ
diff --git a/tests/content/assets/images/foo_tag_cover.jpg b/tests/content/assets/images/foo_tag_cover.jpg
deleted file mode 120000
index 22ce499..0000000
--- a/tests/content/assets/images/foo_tag_cover.jpg
+++ /dev/null
@@ -1 +0,0 @@
-cover.jpg
\ No newline at end of file
diff --git a/tests/content/assets/images/foo_tag_cover.jpg b/tests/content/assets/images/foo_tag_cover.jpg
new file mode 100644
index 0000000..e43b6d1
Binary files /dev/null and b/tests/content/assets/images/foo_tag_cover.jpg differ
diff --git a/tests/content/assets/images/header_cover.jpg b/tests/content/assets/images/header_cover.jpg
deleted file mode 120000
index 22ce499..0000000
--- a/tests/content/assets/images/header_cover.jpg
+++ /dev/null
@@ -1 +0,0 @@
-cover.jpg
\ No newline at end of file
diff --git a/tests/content/assets/images/header_cover.jpg b/tests/content/assets/images/header_cover.jpg
new file mode 100644
index 0000000..e43b6d1
Binary files /dev/null and b/tests/content/assets/images/header_cover.jpg differ
diff --git a/tests/content/assets/images/og_cover.jpg b/tests/content/assets/images/og_cover.jpg
deleted file mode 120000
index 22ce499..0000000
--- a/tests/content/assets/images/og_cover.jpg
+++ /dev/null
@@ -1 +0,0 @@
-cover.jpg
\ No newline at end of file
diff --git a/tests/content/assets/images/og_cover.jpg b/tests/content/assets/images/og_cover.jpg
new file mode 100644
index 0000000..e43b6d1
Binary files /dev/null and b/tests/content/assets/images/og_cover.jpg differ
diff --git a/tests/content/assets/images/page_cover.jpg b/tests/content/assets/images/page_cover.jpg
deleted file mode 120000
index 22ce499..0000000
--- a/tests/content/assets/images/page_cover.jpg
+++ /dev/null
@@ -1 +0,0 @@
-cover.jpg
\ No newline at end of file
diff --git a/tests/content/assets/images/page_cover.jpg b/tests/content/assets/images/page_cover.jpg
new file mode 100644
index 0000000..e43b6d1
Binary files /dev/null and b/tests/content/assets/images/page_cover.jpg differ
diff --git a/tests/content/assets/images/raj_author_cover.jpg b/tests/content/assets/images/raj_author_cover.jpg
deleted file mode 120000
index 22ce499..0000000
--- a/tests/content/assets/images/raj_author_cover.jpg
+++ /dev/null
@@ -1 +0,0 @@
-cover.jpg
\ No newline at end of file
diff --git a/tests/content/assets/images/raj_author_cover.jpg b/tests/content/assets/images/raj_author_cover.jpg
new file mode 100644
index 0000000..e43b6d1
Binary files /dev/null and b/tests/content/assets/images/raj_author_cover.jpg differ
